config/environment.js

```diff
@@ -20,6 +20,10 @@ module.exports = function(environment) {
     APP: {
       // Here you can pass flags/options to your application instance
       // when it is created
+    },
+
+    CashNodes: {
+      baseUrl: 'https://cashnodes-api.bitcoinunlimited.info'
```

Comment:
just a nitt, a curiosity if you wiil, why `baseUrl` rather than `apiUrl` or something like that?